lib/linguist/languages.yml

```
@@ -3642,6 +3642,14 @@ Just:
   - ".just"
   ace_mode: text
   language_id: 128447695
+KDL:
+  type: data
+  color: "#ffb3b3"
+  extensions:
+  - ".kdl"
+  tm_scope: source.kdl
+  ace_mode: text
```

The [ACE](https://ace.c9.io/tool/mode_creator.html) and [CodeMirror modes](https://codemirror.net/5/mode/yacas/index.html) I suggested earlier do a decent job of highlighting KDL source, though their inclusion is purely optional:
```suggestion
  ace_mode: tcl
  codemirror_mode: yacas
  codemirror_mime_type: text/x-yacas
```

ACE and CodeMirror are third-party, in-browser text-editor components that're indirectly supported by Linguist. Because the languages they support are outside our control, we sometimes allow new languages to (ab)use the ACE/CM modes of other languages if they do a decent enough job of highlighting (which ACE's Tcl mode and CodeMirror's YACAS mode do, respectively).

As I said, these inclusions are optional, albeit recommended. It's up to you, however.

---

All done!